src/js/control_plugins/starRating.js

```diff
@@ -36,15 +36,32 @@ window.fbControls.push(function(controlClass) {
      * @return {Object} DOM Element to be injected into the form.
      */
     build() {
-      return this.markup('span', null, {id: this.config.name});
+      this.inputId = this.config.name;
+      let defaultValue = this.config.value || 3.6;
+      this.input = this.markup('input', null, {
+        name: this.inputId,
+        id: this.inputId,
+        value: defaultValue,
+        type: 'hidden'
+      });
+
+      this.field = this.markup('span', null, { id: this.config.name + "_render" });
+
+      return this.markup('div', [this.input, this.field]);
     }
 
     /**
      * onRender callback
      */
     onRender() {
       let value = this.config.value || 3.6;
-      $('#'+this.config.name).rateYo({rating: value});
+      var targetId = this.config.name;
+      $('#' + this.config.name + "_render").rateYo({
```

Comment:
```suggestion
      $(this.field).rateYo({
```
I realize this had extra lookup before but while its being updated we can use `this.field` reference instead to avoid additional jQuery lookup. It will also fix the mix of single and double quotes.